src/hb-coretext.cc

```diff
@@ -341,7 +324,7 @@ hb_coretext_font_data_sync (hb_font_t *font)
   const hb_coretext_font_data_t *data = font->data.coretext;
   if (unlikely (!data)) return nullptr;
 
-  if (fabs (CTFontGetSize((CTFontRef) data) - coretext_font_size_from_ptem (font->ptem)) > .5)
+  if (fabs (CTFontGetSize((CTFontRef) data) - font->ptem) > .5)
```

Comment:
Already inconsistent, but now that you are touching it, please add a space after two `CTFontGetSize` call to make it match with our style. Thanks

Reply:
Done. Also for the call in `hb_coretext_font_create`.